news/2021-08-04-new-featured-artist-3r2.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "New Featured Artist: 3R2"
+date: 2021-08-04 10:00:00 +0000
+---
+
+**3R2** joins the ranks as the newest addition to our Featured Artist lineup!
+
+![](https://assets.ppy.sh/artists/187/header.jpg)
+
+A long-time contributor to all corners of the gaming sphere, **3R2** (a.k.a. **DJ Mashiro**) is a Taiwan-based composer, arranger, and DJ. Anyone in the rhythm game world likely needs no introduction to this creator given he's worked for some of the scene's biggest titles like *Cytus*, *Deemo*, *CHUNITHM*, *Arcaea*, and *Muse Dash*. On the non-rhythm side of gaming, he also produced the soundtracks for *Little With Nobeta* and *Rabi-Ribi*!
```

Comment:
```suggestion
As a long-time contributor to all corners of the gaming sphere, **3R2** (a.k.a. **DJ Mashiro**) is a Taiwan-based composer, arranger, and DJ. Anyone in the rhythm game world likely needs no introduction to this creator given that he's worked for some of the scene's biggest titles like *Cytus*, *Deemo*, *CHUNITHM*, *Arcaea*, and *Muse Dash*. On the non-rhythm side of gaming, he also produced the soundtracks for *Little Witch Nobeta* and *Rabi-Ribi*!
```

Reply:
leaving the first but applied the other 2

Reply:
just preference, so it's alright